client/src/components/main/PostRecipe.jsx

```diff
@@ -34,7 +37,10 @@ class PostRecipe extends Component {
 
   onSubmit(values) {
     const category = this.state.selectedCategory;
```

Comment:
consider making this a an arrow function to avoid explicitly binding the method to the class.
`this.onSubmit = this.onSubmit.bind(this);`

```
onSubmit = (values) => {
 ...
 ...
}
```